dev/README.md

```diff
@@ -29,4 +29,11 @@ $ curl 'localhost:8080/graphql' -H 'Content-Type: application/json' -H 'Accept:
 {"data":{"userLogin":{"authToken":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1aWQiOiI2MTc2YmQ2NmQ0MmFkYWIzNjM2MmEyY2QiLCJuZXR3b3JrIjoibWFpbm5ldCIsImlhdCI6MTYzNTE3MTY4Nn0.n-p5sA9meAmZrVOdngYr216jG3LKOFsFdJmVw6XND3A"}}}
 ```
 
+To test the jupyter lab:
```

Comment:
Can you move this to its own subheading lower in the README.

Reply:
Done.